cobc/ChangeLog

```diff
@@ -4,6 +4,17 @@
 	* codeoptim.c: adjusted inline functions (cob_cmp_packed_int,
 	  cob_get_packed_int) to use register types and less intermediate values
 
+2023-01-28  Fabrice Le Fessant <fabrice.le_fessant@ocamlpro.com>
+
+	* cobc.c (main): initialize cb_config_text_column to 72 to avoid a
+	  race condition in config files.
+	* pplex.l (ppopen): try to autodetect the format of a file before
```

Comment:
actually it is still after the BOM check and that is mentioned directly two four lines below !?!

Reply:
Oups, actually, I did it completely wrong the first time, I wanted to do the two tests independantly, but it's just not possible because you have to do the autodetect after the BOM test, but at the end, you still need to be careful when rewinding not to go back before the BOM...

Reply:
yes, the current code is fine now (better would be only if we would not peek but read and unput, this way it could also be done for code from stdin; but that could be done later, too)
I'll check the ChangeLog again once this leaves draft state.